CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing to IDSA Github
+
+This is the official repository of [IDSA](https://www.internationaldataspaces.org) to provide a general overview of what is available on the IDS Open Source Landscape.
+
+All content published here is provided and approved by the IDSA Head Office. You are very welcome to contribute to this repository when you find a bug, want to suggest an improvement, or have an idea for a useful feature. For this, always create an issue and a corresponding branch, and follow our style guides as described below.
+
+Please note that we have a [code of conduct](CODE_OF_CONDUCT.md) that all contributors should stick to.
+
+
+## Issues
+
+You always have to create an issue if you want to propose a correction, improvement, or feature.
```

Comment:
We might want to elaborate here more on the different things someone can contribute. A bug report is maybe not the only thing. 
- The idea here is to provide knowledge that can be useful for others, too. This could be question (raise a issue) or contribute something that you might have learned, raise an issue and create a branch